stablehlo/reference/CMakeLists.txt

```diff
@@ -14,15 +14,7 @@
 
 add_mlir_library(StablehloReferenceAxes
   PARTIAL_SOURCES_INTENDED
-  Sizes.cpp
```

Comment:
Let's also delete Sizes.cpp and Sizes.h?

Reply:
Done.